front-end-master/common_files/jquery-ui-1.12.1/desktop.ini

```diff
@@ -2,4 +2,3 @@
 InfoTip=This folder is shared online.
 IconFile=C:\Program Files\Google\Drive\googledrivesync.exe
```

Comment:
Should the desktop.ini file be committed? Even if yes, the hardcoded path might pose a problem when we deploy to firebase as the path won't match

Reply:
Sorry i didn't notice that file.